applications/trunkstore/src/ts_from_offnet.erl

```diff
@@ -306,6 +317,7 @@ routing_data(ToDID, AcctID, Settings) ->
                          ,{<<"To-IP">>, build_ip(ToIP, ToPort)}
                          ,{<<"Route-Options">>, RouteOpts}
                          ,{<<"Hunt-Account-ID">>, HuntAccountId}
+                         ,{<<"Authorizing-ID">>, wh_doc:id(Settings)} % connectivity doc id
```

Comment:
@jamesaimonetti shouldn't this be going in `<<"Custom-Channel-Vars">>` ? if so, maybe also add `  ,{<<"Authorizing-Type">>, <<"sys_info">>}`